package.json

```diff
@@ -60,7 +60,6 @@
     "prop-types": "^15.7.2",
     "ramda": "^0.26.1",
     "react-addons-css-transition-group": "^15.6.2",
-    "react-truncate": "^2.4.0",
```

Comment:
Good stuff! This project hasn't been updated in a while [and has some issues with breaking text](https://github.com/One-com/react-truncate/issues/16).